site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -16,17 +16,102 @@
         {% endif %}
 
         <div id="save_status_container" class="customization_item">
-            <span id="status_label">Status: </span>
+            <div id="build_buttons">
+                <button
+                    id="btn-build-customization"
+                    class="btn btn-primary"
+                    tabindex="0"
+                    type="button"
+                    onclick="runBuild()"
+                >
+                    <i class="fas fa-magic-wand-sparkles" title="Build customization"></i>
+                    Build
+                </button>
+            </div>
+            <span id="status_label" style="padding-right: 4px;">Status: </span>
             <div id="save_status" data-testid="save-status">
                 No changes to save
             </div>
             <div id="save_status_buttons">
-                <input id="save_status_button" class="key_to_click" tabindex="0" type="button" value="Save Changes" onclick="ajaxUpdateJSON();" data-testid="save-status-button"/>
-                <input id="show_log_button" type="button" value="Log" />
+                <button style="margin-left: 60px; padding: 5px 12px;" class="btn btn-default" id="show_log_button" type="button">
+                    Log
+                </button>
```

Comment:
Why does this button need special padding?

Reply:
Because I wanted the button size to exactly match Build button, or my center align will kinda downshift the characters.